### PR TITLE
gitsigns no longer requires plenary

### PR DESCRIPTION
**Pull request by andrewferrier**

### CODE REVIEW

**Comment by tjdevries** [comment]
Sorry, there is just some whitespace conflicts. If you could fix, that'd be great and then I'll merge.

**Comment by andrewferrier** [comment]
Thanks. OK, I think this is good to go now.

File: init.lua [hunk 1]
```diff
@@ -9,23 +9,23 @@ end
...
+  use 'neovim/nvim-lspconfig'                                            -- Collection of configurations for built-in LSP client
+  use 'williamboman/mason.nvim'                                          -- Manage external editor tooling i.e LSP servers
+  use 'williamboman/mason-lspconfig.nvim'                                -- Automatically install language servers to stdpath
+  use { 'hrsh7th/nvim-cmp', requires = { 'hrsh7th/cmp-nvim-lsp' } }      -- Autocompletion
+  use { 'L3MON4D3/LuaSnip', requires = { 'saadparwaiz1/cmp_luasnip' } }  -- Snippet Engine and Snippet Expansion
+  use 'mjlbach/onedark.nvim'                                             -- Theme inspired by Atom
+  use 'nvim-lualine/lualine.nvim'                                        -- Fancier statusline
+  use 'lukas-reineke/indent-blankline.nvim'                              -- Add indentation guides even on blank lines
+  use 'tpope/vim-sleuth'                                                 -- Detect tabstop and shiftwidth automatically
+
```
Comment (jbranchaud) [inline init.lua:28]: Removes superfluous spaces on newline

```suggestion

```

**Comment by tjdevries** [comment]
Sorry, I ended up just fixing this after reformatting everything. Thanks for the PR :) it was helpful!

**Pull request closed without merging**